dpgen2/exploration/task/lmp_spin_task_group.py

```diff
@@ -0,0 +1,112 @@
+import itertools
+import random
+from pathlib import (
+    Path,
+)
+from typing import (
+    List,
+    Optional,
+)
+
+from dpgen2.constants import (
+    lmp_conf_name,
+    lmp_input_name,
+    lmp_traj_name,
+    model_name_pattern,
+    plm_input_name,
+    plm_output_name,
+)
+
+from .conf_sampling_task_group import (
+    ConfSamplingTaskGroup,
+)
+from .lmp import (
+    make_lmp_input,
+)
+from .task import (
+    ExplorationTask,
+)
+
+
+class LmpSpinTaskGroup(ConfSamplingTaskGroup):
+    def __init__(
+        self,
+    ):
+        super().__init__()
+        self.lmp_set = False
+        self.plm_set = False
+
+    def set_lmp(
+        self,
+        numb_models: int,
+        lmp_template_fname: str,
+        plm_template_fname: Optional[str] = None,
+        revisions: dict = {},
+    ) -> None:
+        self.lmp_template = Path(lmp_template_fname).read_text().split("\n")
+        self.revisions = revisions
+        self.lmp_set = True
+        self.model_list = sorted([model_name_pattern % ii for ii in range(numb_models)])
```

Comment:
What's the difference between a spin task group and a normal template task group? It seems that the spin task group does not revise input models in the template.